buildSrc/src/main/kotlin/io.customer/android/Dependencies.kt

```diff
@@ -29,6 +29,7 @@ object Dependencies {
     const val junit4 = "junit:junit:${Versions.JUNIT4}"
     const val kluent = "org.amshove.kluent:kluent:${Versions.KLUENT}"
     const val kotlinGradlePlugin = "org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.KOTLIN}"
+    const val lifecycleCommon = "androidx.lifecycle:lifecycle-common:${Versions.LIFECYCLE_COMMON}"
```

Comment:
Do we need this dependency? Can we use [this](https://developer.android.com/reference/android/app/Application.ActivityLifecycleCallbacks) instead?

Reply:
`DefaultLifecycleObserver` needs this, but it's focused more on the latest API. We can ignore for now.